packages/runtime-node/src/communication-helpers.ts

```diff
@@ -0,0 +1,65 @@
+import { ChildProcess, Serializable } from 'node:child_process';
+import { type MessagePort, type Worker } from 'node:worker_threads';
+import { deferred } from 'promise-assist';
+
+/**
+ * Emits an event using specified emitter as MessagePort
+ */
+export function emitEvent<T extends Serializable>(
+    emitter: MessagePort | Worker | NodeJS.Process | ChildProcess,
```

Comment:
not sure we want to combine it all to a single fn.
some of these require json serializable data, while others don't.

Reply:
removed these helpers